### DISCUSSION TITLE
Text Field Default Value with State Doesn't Work

### DISCUSSION THREAD

**Discussion by sjarifHD** [Budibase/budibase#5362] [Help]
I has been updated Budibase self host from v1.0.98 to v1.0.110, after got an error when access published apps, I recreate every apps with export and import. And then when the problem solved for access published apps, another problem happen with state in budibase.

My apps usually using state for pre-filled text-field in form using default value. 

So for example, I have text-field for searching Customer (picture 1). When I click the button in table usually in v1.0.98 below it can set default value for text-field. But now, it doesn't work anymore. I try to read state in text-field label, strangely it works (picture 2).
![image](https://user-images.githubusercontent.com/9961361/162935652-2f2b0fbb-e873-49d8-a649-03a1f53bd249.png)
![image](https://user-images.githubusercontent.com/9961361/162936210-59060971-b4ff-4ae4-9a79-f6e1d3c2ccb3.png)

_NB: The form using Indonesian language_

**Comment by melohagan** [comment]
Hi @sjarifHD 

The default value behaviour has now changed so that the it is not updatable via state.
Mostly this was done to resolve issues with infinite loops; `Date.now()` for example.

What you could do is pass the default name as a URL parameter into your screen (similar to the service name here: https://github.com/Budibase/budibase/discussions/5299#discussioncomment-2530261)

Then your button, instead of (or in addition to) updating state, it could 'Navigate To' `/farmers/<-name-here->`
